.upstream-tests/test/std/utilities/function.objects/comparisons/constexpr_init.pass.cpp

```diff
@@ -8,7 +8,7 @@
 
 // UNSUPPORTED: c++98, c++03, c++11
 
-// XFAIL: gcc-7
+// XFAIL: gcc-7 && !nvrtc
```

Comment:
Just to be sure about the syntax, the other test exclusion comments are comma separated lists.